src/handlers.ts

```diff
@@ -764,6 +768,21 @@ export function createDerepostActionHandler(accountService: AccountService) {
 
         const actor = await apCtx.getActor(ACTOR_DEFAULT_HANDLE); // TODO This should be the actor making the request
 
+        if (!post.id) {
```

Comment:
I think it's slightly better to use the `id` parsed in from params here, because when we eventually pull the fedify stuff out of this handler, there will be no `post` instance to use the `id` of!

It's not a big deal, can always fix it then